tfe_integration_test.go

```diff
@@ -27,6 +27,7 @@ func TestClient_newClient(t *testing.T) {
 		w.Header().Set("Content-Type", "application/vnd.api+json")
 		w.Header().Set("X-RateLimit-Limit", "30")
 		w.Header().Set("TFP-API-Version", "34.21.9")
+		w.Header().Set("X-TFE-Version", "202205-1")
```

Comment:
What do you think about testing for the case where we have TFC?

Reply:
The issue with this is that I'd have rewrite most of the test simply to test that a field is an empty string (the test isn't really operating under a TFC vs TFE context).  

One nice guarantee we have is that the function that's used to fetch the header `http.Response.Header.Get()` returns a string and the docs mention that if a key does not exist, it will return an empty string. So by testing that we can fetch the value of that header we are also inadvertently testing that it can potentially be empty as well since we know that header can be omitted.